addons/friendly_tracker/functions/fnc_stop.sqf

```diff
@@ -0,0 +1,32 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3, veteran29
+ * Handles termination of BFT
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call afm_friendly_tracker_fnc_stop
+ *
+ * Public: No
+ */
+
+if (hasInterface && {!GVAR(enabled)} && {GVAR(running)}) exitWith {
```

Comment:
Is there a point to check if `hasInterface` if the whole thing can't be started without it?

Consider doing `if (!GVAR(running)) exitWith {};` to reduce nesting, no point to check for `enabled` as it was done in `loop`.

Reply:
Yeah, unnecessary redundancy. Just checking `GVAR(running)` is enough.